rust_icu_sys/bindgen/run_bindgen.sh

```diff
@@ -0,0 +1,37 @@
+#! /bin/bash
+# Copyright 2019 The Fuchsia Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+#
+# This is a manual update file for bindgen.
+set -euo pipefail
+set -x
+
+# Types for which to generate the bindings.  Expand this list if you need more.
+# The syntax is regex.
+readonly GENERATE_TYPES="UBool|UCalendar.*|UChar.*|UData.*|UDate|UDateFormat.*|UEnumeration.*|UErrorCode|UText.*"
+
+# Functions for which to generate the bindings.  Expand this list if you need more.
+readonly GENERATE_FUNCTIONS="u_.*|ucal_.*|udat_.*|udata_.*|uenum_.*|uloc_.*|utext_.*"
```

Comment:
It would be great if there were some way to share these variables with `build.rs`. Barring that, we should have comments reminding developers to keep them in sync.

Reply:
> It would be great if there were some way to share these variables with `build.rs`. Barring that, we should have comments reminding developers to keep them in sync.

Agreed here too.  Can we defer doing that until we have the current code up and running?

Reply:
Sure.